microcosm_flask/generic_resources.py

```diff
@@ -1,8 +0,0 @@
-from marshmallow import Schema, fields
-
-
-class EmptySchema(Schema):
```

Comment:
I'm removing this base; I will take care of removing its usage once this releases.